reference/service/saml-okta.md

```diff
@@ -47,11 +44,12 @@ tbody tr td {
 
 | General Setting | Value |
 | --------------- | ----- |
-| Single sign on URL | https://api.pulumi.com/login/robot-co/sso/saml/acs |
-| Audience URI | https://api.pulumi.com/login/robot-co/sso/saml/metadata |
+| Single sign on URL | https://api.pulumi.com/login/**acmecorp**/sso/saml/acs |
+| Audience URI | https://api.pulumi.com/login/**acmecorp**/sso/saml/metadata |
 | Name ID format | Unspecified |
 | App username | Email |
 
+<p><!-- space between table and text --></p>
```

Comment:
😱 😄  (But we really should be using CSS for this instead. The rework coming up can hopefully address this sort of thing.)

Reply:
Yes, we should. Please forgive this obviously flawed code smell.

I'm intentionally not touching the CSS in hopes that the rework will fix all known problems.